src/components/KPrompt/KPrompt.vue

```diff
@@ -139,6 +146,7 @@ const sanitizedAttrs = computed(() => {
 })
 
 const confirmationInput = ref<string>('')
+const displayErrorState = ref<boolean>(false)
```

Comment:
![image](https://github.com/user-attachments/assets/b09c7082-b29a-4e79-b134-303940ab9ea8)

issue: if the error state is triggered, and then I close the prompt and re-open it, the error state persists. I believe this ref should be reset when closing/canceling

Reply:
Done